i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -145,6 +145,68 @@ Wenn Sie die Möglichkeit sehen, das Passwort in den Einstellungen zu ändern, k
+  <ol>
+    <li>Tippen Sie auf <b>Bildschirm sperren</b>.</li>
+    <li>Geben Sie eine PIN ein, die Sie zum Entsperren verwenden möchten.</li>
+    <li>Bestätigen Sie die PIN.</li>
+    <li>Sie werden nun eine Seite mit verschiedenen Schaltern und Optionen sehen und wann immer Sie die App öffnen, werden Sie nach Ihrer PIN gefragt.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+### Ändern der PIN für die Bildschirmsperre
+
+<Tabs>
+  <TabItem value="mobil" label="Mobil">
+  <ol>
+    <li>Tippen Sie auf <b>PIN-Code ändern</b>.</li>
+    <li>Geben Sie Ihre aktuelle PIN ein.</li>
+    <li>Geben Sie nun Ihre neue PIN ein.</li>
+    <li>Wiederholen Sie die neue PIN.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+### Nutzen von Biometrischer Authentifizierung
+
+Anstelle der PIN können Sie Ihr Gerät auch mittels Biometrie wie z.B. Gesichtserkennung oder Fingerabdruck entsperren. Die Biometrische Authentifizierung wird die Standard-Authentifizierungsmethode Ihres Gerätes verwenden. Sie können auch weiterhin Ihre PIN verwenden, um die Bildschirmsperre aufzuheben, außerdem wird die PIN zum Deaktivieren der Bildschirmsperre benötigt. Falls Sie Ihre PIN vergessen, können Sie die Bildschirmsperre nur noch durch einen Logout deaktivieren. 
+
+<Tabs>
+  <TabItem value="mobil" label="Mobil">
+  <ol>
+    <li>Tippen Sie auf <b>Biometrische Authentifizierung zulassen</b>.</li>
+    <li>Identifizieren Sie sich mit der biometrischen Authentifizierung auf Ihrem Mobilgerät.</li>
+    <li>Wenn Sie nun versuchen, die App zu entsperren, werden Sie aufgefordert, sich mittels Biometrie zu authentifizieren. Sie können diesen Dialog abbrechen und stattdessen Ihre PIN verwenden.</li>
+    <li>Falls Sie den Dialog versehentlich abgebrochen haben oder er nicht angezeigt wird, können Sie auf „Biometrische Authentifizierung verwenden“ tippen, um eine erneute Abfrage der Biometrie anzufordern.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
+### Auto lock zum Sperren des Bildschirms verwenden
+
+Falls Sie sichergehen möchten, dass niemand Ihre Nachrichten lesen kann, z.B. wenn Sie vergessen, die App in den Hintergrund zu versetzen, können Sie zwischen verschiedenen Zeitspannen wählen, nach denen die App automatisch gesperrt wird. 
+
+<Tabs>
+  <TabItem value="mobil" label="Mobil">
+  <ol>
+    <li>Tippen Sie auf <b>Auto lock</b>.</li>
+    <li>Wählen Sie nun die Zeitspanne aus der Liste, nach der die App sich sperren soll.</li>
+    <li>Die App wird sich nun jedes Mal wenn die gewählte Zeitspanne verstreicht automatisch sperren.</li>
+    <li>Falls Sie das Auto Lock Feature nicht mehr nutzen wollen, tippen Sie erneut auf <b>Auto lock</b> und wählen Sie „Deaktivieren“ aus der Liste aus.</li>
+  </ol>
+  </TabItem>
+</Tabs>
+
 ## Verbundene Geräte
 
 Hier finden Sie alle Geräte die mit Ihrem Gerät verbunden sind. Sie können die Geräte sperren und entfernen. Bitte beachten Sie, dass Sie Zugriff auf den Geräten verlieren, wenn Sie diese ausloggen. Sie müssen sich auf diesen Geräten dann neu einloggen.
```

Comment:
Was ist der Unterschied zwischen sperren und entfernen?

Reply:
This is not part of this Pull request since I didn't change anything here.
I think the difference between blocking and deleting devices is that a device that's deleted will get logged out on deletion whereas a blocked device just isn't a trusted device anymore, I think. Never used the blocking feature myself though.